packages/components/src/components/counter/counter.tsx

```diff
@@ -0,0 +1,11 @@
+import classNames from 'classnames';
+import React, { HTMLProps } from 'react';
+
+type TCounterProps = HTMLProps<HTMLDivElement> & {
+    count: number;
+};
```

Comment:
```suggestion
type TCounterProps = {
    className?: string;
    count: number;
};

```
@iman-fs By extending `HTMLProps<HTMLDivElement>` we are telling TypeScript we accept all those props and whoever is going to use `Counter` will be able to pass them and get autocomplete for them, But inside the component, the only valid prop is `className`.